src/main/groovy/com/twcable/grabbit/jcr/JCRNodeDecorator.groovy

```diff
@@ -105,6 +106,26 @@ class JCRNodeDecorator {
         return definition.isMandatory()
     }
 
+    /**
+     * Returns the "jcr:created", "jcr:lastModified" or "cq:lastModified" date property
+     * for current Jcr Node
+     * Returns null of none of the 3 are found
```

Comment:
```java
/**
*This method is to determine the last time a node was modified 
*including the situation where it's last modified date is it's creation date
*@return the last time this node was modified if we can determine the date, otherwise return null 
*/
@Nullable
Date getLastModified() {
```